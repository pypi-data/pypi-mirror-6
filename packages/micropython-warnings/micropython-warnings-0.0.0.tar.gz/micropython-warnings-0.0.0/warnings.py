# MicroPython doesn't support warnings
