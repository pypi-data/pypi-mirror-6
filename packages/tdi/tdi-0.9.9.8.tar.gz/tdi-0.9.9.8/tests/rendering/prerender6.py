#!/usr/bin/env python
import warnings as _warnings
_warnings.resetwarnings()
_warnings.filterwarnings('error')

from tdi.model_adapters import RenderAdapter
from tdi import html

template = html.from_string("""
<anode tdi="anode"></anode>
""".lstrip())

class Model(object):
    def __init__(self, version):
        self._version = version

    def prerender_version(self, version):
        return version != self._version, self._version

    def render_anode(self, node):
        print "render_anode", self._version


print template.render_string(prerender=Model(1))
print "---"
print template.render_string(prerender=Model(1))
print "---"
print template.render_string(prerender=Model(2))
print "---"
print template.render_string(prerender=Model(2))
