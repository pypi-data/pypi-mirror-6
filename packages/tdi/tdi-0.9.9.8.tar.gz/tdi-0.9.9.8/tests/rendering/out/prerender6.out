render_anode 1
<anode></anode>

---
<anode></anode>

---
render_anode 2
<anode></anode>

---
<anode></anode>

